### PR TITLE
pymummer: new formula 0.4.0

### PR DESCRIPTION
**Pull request by andrewjpage**
Pymummer is a python library for the bioinformatics tool [MUMmer](http://mummer.sourceforge.net/).

### CODE REVIEW

File: pymummer.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  desc "Python3 wrapper for running MUMmer and parsing the output."
+  homepage "https://github.com/sanger-pathogens/pymummer"
+  url "https://github.com/sanger-pathogens/pymummer/archive/v0.4.0.tar.gz"
+  sha256 "900b25019e5351882ea62c5992f019903cb838a2c80375de3842159e04cf1588"
+  head "https://github.com/sanger-pathogens/pymummer.git"
+  # tag "bioinformatics"
+
+  depends_on :python3
+  depends_on "homebrew/science/mummer"
+  depends_on "homebrew/python/numpy" => ["without-python", "with-python3"]
```
Comment (tdsmith) [inline pymummer.rb]: Just "numpy" since this is in homebrew-python, and don't spec "without-python".

File: pymummer.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    %w[pyfastaq].each do |r|
+      resource(r).stage do
+        system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+        pyfastaq_path = libexec/"vendor/lib/python#{version}/site-packages"
+        dest_path = lib/"python#{version}/site-packages"
+        mkdir_p dest_path
+        (dest_path/"homebrew-pymummer-pyfastaq.pth").write "#{pyfastaq_path}\n"
+      end
+    end
```
Comment (tdsmith) [inline pymummer.rb]: You can get rid of the `each` and just do `resource("pyfastaq").stage...`

File: pymummer.rb (hunk 1 above)
Comment (tdsmith) [inline pymummer.rb]: (same for scipy and pillow)

File: pymummer.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  depends_on "homebrew/python/pillow" => ["without-python", "with-python3"]
+
+  resource "pyfastaq" do
+    url "https://pypi.python.org/packages/source/p/pyfastaq/pyfastaq-3.5.0.tar.gz"
+    sha256 "599b28db5a05072335eacd3cea458aff511239e34a9559aaf2e7fe94cce785a9"
+  end
+
+  def install
+    version = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{version}/site-packages"
```
Comment (tdsmith) [inline pymummer.rb]: This line looks unnecessary since you aren't installing anything there.

**Comment by andrewjpage** [comment]
I've made the changes you've suggested, thanks for the feedback. Tested it on Ubuntu Trusty.

**Comment by tdsmith** [comment]
Pulled in 17f879ff2a. Thanks for the PR!

**Pull request closed without merging**